project.json

```diff
@@ -2,21 +2,21 @@
   "deps": [
     {
       "name": "wroboclaw",
-      "repo": "https://github.com/WisconsinRobotics/wroboclaw.git",
+      "repo": "ssh://git@wroboclaw.github.com/WisconsinRobotics/wroboclaw.git",
       "build": {
         "provider": "pip"
       }
     },
     {
       "name": "wready",
-      "repo": "https://github.com/WisconsinRobotics/wready.git",
+      "repo": "ssh://git@wready.github.com/WisconsinRobotics/wready.git",
       "build": {
         "provider": "pip"
       }
     },
     {
       "name": "wreadinput",
-      "repo": "https://github.com/WisconsinRobotics/wreadinput.git",
+      "repo": "ssh://git@wreadinput.github.com/WisconsinRobotics/wreadinput.git",
       "build": {
         "provider": "pip"
       }
```

Comment:
These changes should probably not be here, since they will break the build for developers not on the base station or wrover.  For the time being, public checkout is fine; maybe this can be moved to a script or alternate file for setup on the competition system.

Reply:
Addressed in 9be248d.